server/Mac/mf_event.h

```diff
@@ -18,7 +18,6 @@
  */
 
 #ifndef __MF_EVENT_H
-#define __XMF_EVENT_H
```

Comment:
-1, this is a header include guard! I'll agree that the form is bad (__ is reserved for compiler internal use) so maybe something like ```#ifnded MF_EVENT_H #define MF_EVENT_H```